samples/Calculator/src/Calculator.Console/Program.cs

```diff
@@ -0,0 +1,34 @@
+﻿using Calculator.Core;
+
+internal class Program
+{
+    private static int Main(string[] args)
+    {
+        if (args.Length != 3)
+        {
+            PrintHelp();
+            return -1;
+        }
+
+        string operation = args[0].ToLower();
+        double a = Convert.ToDouble(args[1]);
+        double b = Convert.ToDouble(args[2]);
+
+        Console.WriteLine(OperationCalculator.Calculate(operation, a, b));
+        return 0;
+    }
+
+    private static void PrintHelp()
+    {
+        Console.WriteLine(Help);
+    }
+
+    private const string Help = @"Invalid usage
```

Comment:
```suggestion
    private const string Help = "Invalid usage
```

Reply:
Or 
```
"""
"""
```

Reply:
I have newline there and both " and """ are not working. Maybe new language features are not enabled